### PR TITLE
[16.0] [MIG] partner_firstname

### PR DESCRIPTION
**Pull request by baimont**
https://github.com/OCA/partner-contact/issues/1353

### CODE REVIEW

**Review by FrancoMaxime — APPROVED** [review]

File: partner_firstname/tests/test_user_form.py [hunk 1]
```diff
@@ -48,17 +48,4 @@ def test_create_from_form_all(self):
         self.assertEqual(user_form.name, " ".join((firstname, lastname)))
 
     def get_clean_user_form(self):
```
Comment (PierrickBrun) [inline partner_firstname/tests/test_user_form.py]: maybe this should be named `_get_user_form`, now that there is no cleaning

**Review by PierrickBrun — APPROVED** [review]
code review, LGTM

ping @carmenbianca

File: partner_firstname/tests/test_user_form.py (hunk 1 above)
Comment (baimont) [reply partner_firstname/tests/test_user_form.py]: done

**Review by marielejeune — APPROVED** [review]
Code review done

**Review by carmenbianca — APPROVED** [review]
LGTM

**Pull request merged**